lib/npm-check-updates.js

```diff
@@ -97,20 +97,54 @@ function createDependencyTable() {
             middle: ''
         }
     });
+    return baseTable;
+}
+
+/**
+ * @param {object} args
+ * @param {object} options
+ * @returns {Promise<object>}
+ */
+async function getOwnerPerDependency(args, options) {
+    const packageManager = vm.getPackageManager(options.packageManager);
+    args.owner = {};
+    await Promise.all(
+        Object.keys(args.to).map(async (dep) => {
+            const from = args.from[dep] || null;
+            const to = args.to[dep] || null;
+            const ownerChanged = await packageManager.packageAuthorChanged(dep, from, to);
+            // eslint-disable-next-line require-atomic-updates
+            args.owner[dep] = ownerChanged;
+        })
+    );
+    return args;
 }
 
 /**
  * @param {object} args
  * @param {object} args.from
  * @param {object} args.to
- * @returns {Table}
+ * @param {object} args.owner
+ * @param {object} options
+ * @returns {Promise<Table>}
  */
-function toDependencyTable(args) {
+async function toDependencyTable(args, options) {
     const table = createDependencyTable();
-    const rows = Object.keys(args.to).map(dep => {
-        const from = args.from[dep] || '';
-        const to = versionUtil.colorizeDiff(args.from[dep], args.to[dep] || '');
-        return [dep, from, '→', to];
+    let mappedArgs = [];
+    if (options.ownerChanged) {
+        const newArgs = await getOwnerPerDependency(args, options);
+        mappedArgs = _.cloneDeep(newArgs);
+    } else {
+        mappedArgs = _.cloneDeep(args);
+    }
+    const rows = Object.keys(mappedArgs.to).map(dep => {
+        const from = mappedArgs.from[dep] || '';
+        const to = versionUtil.colorizeDiff(mappedArgs.from[dep], mappedArgs.to[dep] || '');
+        let owner = '';
+        if (options.ownerChanged) {
+            owner = mappedArgs.owner && mappedArgs.owner[dep] ? '*owner changed*' : '';
```

Comment:
The `if` statement could be simplified since you are already using a `ternary`:

```js
const owner = options.ownerChanged && mappedArgs.owner && mappedArgs.owner[dep] ? '*owner changed*' : '';
```